examples/acados_matlab_octave/getting_started/extensive_example_ocp.m

```diff
@@ -296,60 +293,15 @@
 
 %% get QP matrices:
 % See https://docs.acados.org/problem_formulation
+%        |----- dynamics -----|------ cost --------|---------------------------- constraints ------------------------|        
+fields = {'qp_A','qp_B','qp_b','qp_R','qp_Q','qp_r','qp_C','qp_D','qp_lg','qp_ug','qp_lbx','qp_ubx','qp_lbu','qp_ubu'};
 % either stage wise
-for stage = [0, N-1]
-    % Note loop over field doesnt work because stupid matlab diff between
-    % chars and strings
-    % dynamics
-    field = 'qp_A';
-    disp(strcat(field, " at stage ", num2str(stage), " = "));
-    ocp.get(field, stage)
-    field = 'qp_B';
-    disp(strcat(field, " at stage ", num2str(stage), " = "));
-    ocp.get(field, stage)
-    field = 'qp_b';
-    disp(strcat(field, " at stage ", num2str(stage), " = "));
-    ocp.get(field, stage)
-
-    % cost
-    field = 'qp_R';
-    disp(strcat(field, " at stage ", num2str(stage), " = "));
-    ocp.get(field, stage)
-    field = 'qp_Q';
-    disp(strcat(field, " at stage ", num2str(stage), " = "));
-    ocp.get(field, stage)
-    field = 'qp_r';
-    disp(strcat(field, " at stage ", num2str(stage), " = "));
-    ocp.get(field, stage)
-
-    % constraints
-    field = 'qp_C';
-    disp(strcat(field, " at stage ", num2str(stage), " = "));
-    ocp.get(field, stage)
-    field = 'qp_D';
-    disp(strcat(field, " at stage ", num2str(stage), " = "));
-    ocp.get(field, stage)
-
-    field = 'qp_lg';
-    disp(strcat(field, " at stage ", num2str(stage), " = "));
-    ocp.get(field, stage)
-    field = 'qp_ug';
-    disp(strcat(field, " at stage ", num2str(stage), " = "));
-    ocp.get(field, stage)
-
-    field = 'qp_lbx';
-    disp(strcat(field, " at stage ", num2str(stage), " = "));
-    ocp.get(field, stage)
-    field = 'qp_ubx';
-    disp(strcat(field, " at stage ", num2str(stage), " = "));
-    ocp.get(field, stage)
-
-    field = 'qp_lbu';
-    disp(strcat(field, " at stage ", num2str(stage), " = "));
-    ocp.get(field, stage)
-    field = 'qp_ubu';
-    disp(strcat(field, " at stage ", num2str(stage), " = "));
-    ocp.get(field, stage)
+for stage = 0:N-1
```

Comment:
I would keep it to stage = [0, N-1] here, since the output is a bit much otherwise :)

Reply:
Agreed, reverted :)